components/panelbar/templates/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Templates
+page_title: PanelBar - Templates Overview
+description: Templates in the PanelBar for Blazor.
+slug: panelbar-templates
+tags: telerik,blazor,panelbar,templates
+published: True
+position: 1
+---
+
+# PanelBar Templates
+
+The PanelBar component allows you to define custom templates for both the parent and children items.
+
+The PanelBar can use templates for:
+
+* You can control the rendering of the [Header items]({%slug panelbar-overview%}#elements-of-a-panelbar-item) by using the [HeaderTemplate]({%slug panelbar-templates-header%})
+
+* You can control the rendering of the [Header items]({%slug panelbar-overview%}#elements-of-a-panelbar-item) by using the [ContentTemplate]({%slug panelbar-templates-content%})
```

Comment:
"Content" items may be?

Reply:
the overview article has a section - Elements of a PanelBar Item which explains the header and content items. That said, I have made a typo where the second bullet states "Header items" instead of "Content items" - I have fixed it.